<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LPCXpresso Burner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="6">
     <widget class="QPushButton" name="pushButton_link">
      <property name="text">
       <string>Check LPC-Link</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="pushButton_boot">
      <property name="text">
       <string>Boot LPC</string>
      </property>
     </widget>
    </item>
    <item row="7" column="10">
     <widget class="QPushButton" name="pushButton_run">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="10">
     <widget class="QLineEdit" name="lineEdit_file">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_image">
      <property name="text">
       <string>Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="4" colspan="8">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="8">
     <widget class="QPushButton" name="pushButton_burn">
      <property name="text">
       <string>Burn Image</string>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="7">
     <widget class="QPushButton" name="pushButton_target">
      <property name="text">
       <string>Check Target</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="radioButton_winusb">
      <property name="text">
       <string>winusb</string>
      </property>
     </widget>
    </item>
    <item row="4" column="11">
     <widget class="QPushButton" name="pushButton_file">
      <property name="text">
       <string>File ...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="9">
     <spacer name="horizontalSpacer_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QCheckBox" name="checkBox_multiple">
      <property name="text">
       <string>Burn Multiple Targets Automatically</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_driver">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Driver</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButton_hid">
      <property name="text">
       <string>hid</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="11">
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="12">
     <widget class="QTextEdit" name="textEdit_command">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>radioButton_hid</tabstop>
  <tabstop>radioButton_winusb</tabstop>
  <tabstop>lineEdit_file</tabstop>
  <tabstop>pushButton_file</tabstop>
  <tabstop>checkBox_multiple</tabstop>
  <tabstop>pushButton_boot</tabstop>
  <tabstop>pushButton_link</tabstop>
  <tabstop>pushButton_target</tabstop>
  <tabstop>pushButton_burn</tabstop>
  <tabstop>pushButton_run</tabstop>
  <tabstop>pushButton_stop</tabstop>
  <tabstop>textEdit_command</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
